server/models/service-instance.js

```diff
@@ -84,6 +84,158 @@ module.exports = function extendServiceInstance(ServiceInstance) {
   }
   ServiceInstance.recordInstanceInfo = recordInstanceInfo;
 
+  function runCommand(req, callback) {
+    this.actions.create({
+      request: req
+    }, function(err, action) {
+      if (err) return callback(err);
+      if (action.result && action.result.error)
+        return callback(Error(action.result.error));
+
+      callback(null, action.result);
+    });
+  }
+  ServiceInstance.prototype.runCommand = runCommand;
+
+  function _appCommand(obj, callback) {
+    obj.sub = obj.cmd;
+    obj.cmd = 'current';
+    this.runCommand(obj, callback);
+  }
+  ServiceInstance.prototype._appCommand = _appCommand;
```

Comment:
This appears to be used extensively by ServiceProcess internals, perhaps the `_` prefix should be removed?